src/games/strategy/engine/framework/headlessGameServer/AvailableGames.java

```diff
@@ -164,18 +167,9 @@ private static boolean addToAvailableGames(final URI uri, final Map<String, URI>
           }
           return true;
         }
-      } catch (final Exception e2) {// ignore
```

Comment:
We can simplify the exception handling to be just one try/catch around `new GameParser().parse(`, since we now handle and log IOException in  `UrlStreams.openStream(uri)` (line 157).  This means that the if check for an input stream: `if (inputStream.isPresent()) {`, is the split between when we would get an IOException because we could not read from the input source, vs any exceptions that would occur while reading.